modules/afsocket/transport-mapper-inet.h

```diff
@@ -36,6 +36,7 @@ typedef struct _TransportMapperInet
   gboolean require_tls;
   gboolean allow_tls;
   gboolean require_tls_when_has_tls_context;
+  gboolean proxied;
```

Comment:
Add OneIdentity copyright to the file.
The correct copyright today is: `Copyright (c) 2024 One Identity LLC.`